cpp/tests/test_odesecir.cpp

```diff
@@ -572,10 +574,14 @@ TEST(TestOdeSecir, testModelConstraints)
 
     model.parameters.get<mio::osecir::IncubationTime>()[(mio::AgeGroup)0]       = 5.2;
     model.parameters.get<mio::osecir::TimeInfectedSymptoms>()[(mio::AgeGroup)0] = 5;
-    model.parameters.get<mio::osecir::SerialInterval>()[(mio::AgeGroup)0]       = 4.2;
+    model.parameters.get<mio::osecir::SerialInterval>()[(mio::AgeGroup)0]       = 3.8;
```

Comment:
```suggestion
    model.parameters.get<mio::osecir::SerialInterval>()[(mio::AgeGroup)0]       = 3.9;
```
In the apply/check_constraints() function, we define a allowed interval for the serial interval using the incubation time. The mid of the interval would be 3.9 assuming the incubation time is set as above.
However the problem here is our sampling procedure which often maps values to the allowed edges.